examples_tf2_py/examples_tf2_py/async_waits_for_transform.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
```

Comment:
I think we typically eschew the shebang nowadays, correct?  In that case, please remove it.  (the same comment applies several other times below)

Reply:
Replaced shebang with copyright comment in 48ca46e4c1a7d81d13799d725a0fbdeafbbb7cac